app/presenters/content_item_presenter.rb

```diff
@@ -0,0 +1,48 @@
+class ContentItemPresenter
+  attr_reader :content_item
+
+  def initialize(content_item)
+    @content_item = content_item
+  end
+
+  PASS_THROUGH_KEYS = [
+    :title, :details, :base_path, :description
+  ].freeze
+
+  PASS_THROUGH_DETAILS_KEYS = [
+    :body
+  ].freeze
+
+  PASS_THROUGH_KEYS.each do |key|
+    define_method key do
+      content_item[key.to_s]
+    end
+  end
+
+  PASS_THROUGH_DETAILS_KEYS.each do |key|
+    define_method key do
+      details[key.to_s] if details
+    end
+  end
+
+  def in_beta
+    @content_item['phase'] == 'beta'
```

Comment:
We could add something to the integration test for testing beta content? After a quick glance I don't think we have anything that tests this for help pages (although we do for the other formats)